cmd/controller-manager/app/controller_manager.go

```diff
@@ -143,6 +143,8 @@ func Run(controllerManagerOptions *options.ControllerManagerServer) error {
 		// TODO: disambiguate API errors
 		return fmt.Errorf("failed to get Service Catalog client configuration: %v", err)
 	}
+	// TODO (nilebox): Dirty temporary hack. Should whether accept as flag for application, or handle TLS correctly
```

Comment:
"accept as flag" OR "handle TLS correctly" (by passing certificate authority and other stuff into client configuration). sorry for my poor English if it's confusing you.

This comment is obsolete anyway, I went with the flag for now (see newer commits), as suggested by @pmorie.
We will still need to fix all the places where we pass `--insecure-skip-verify` flags by adding TLS configuration on the client side, but I see this to be a separate PR, and I think we have agreement on this.